bin/travis-ci/conf.vexpress_ca15_tc2_qemu

```diff
@@ -21,7 +21,7 @@
 console_impl=qemu
 qemu_machine="vexpress-a15"
 qemu_binary="qemu-system-arm"
-qemu_extra_args="-nographic -m 1G -tftp /tftpboot"
+qemu_extra_args="-nographic -m 1G -tftp ${UBOOT_TRAVIS_BUILD_DIR}"
```

Comment:
Well, it depends. You can use the old-style -net commands. qemu-system-arm -M vexpress-a15 -net nic -net user,... should give you a similar effect to the -device and -netdev commands, just that device creation happens explicitly in the board file rather than generic bus slot magic.

But this is diverging and going into QEMU internals. All of the variants should work to some extent (except for -device for sysbus devices). It doesn't actually matter which one we use, as long as the guest gets to see the correct network device.